pkg/services/grpcserver/interceptors/auth.go

```diff
@@ -133,6 +135,22 @@ func (a *authenticator) getSignedInUser(ctx context.Context, token string) (*use
 	return signedInUser, nil
 }
 
+func (a *authenticator) getAnonymousUser(ctx context.Context) (*user.SignedInUser, error) {
+	getOrg := org.GetOrgByNameQuery{Name: a.cfg.AnonymousOrgName}
```

Comment:
@ryantxu i based this on code from searchV2. any thoughts on where to put it so that both services could use it?

Reply:
`services/user`? I am pretty sure its not the only place constructing the fake anonymous signedInUser